_assets/javascripts/main.js

```diff
@@ -40,12 +40,9 @@
 				if ($theme === 'dark') {
 					toggleThemeIcon($elToggleTheme);
 				}
-				$window.on('load', function() {

```

Comment:
the point of the `$window.on('load'` is to wait for the page to load before removing the class and now we're just removing the class without waiting. I think this is not the right approach

What do you think @thiamsantos ?

Reply:
@tajchumber This approach will inevitably incur in a race condition, as that is a high chance that the `load` event already happened before the `$window.on('load'` listener even been added. The option I could think of was to add the `is-loading` class directly at the html, with that we could remove the `$window.on('load'` listener from inside the `ready` listener, and be sure that the event will always be triggered. WDYT @tajchumber?

The change would mainly be
- Alter https://github.com/elixirschool/elixirschool/blob/master/_includes/_sidebar.html#L2 to `<div id="sidebar" class="is-loading">`
- Add outside of https://github.com/elixirschool/elixirschool/blob/master/_assets/javascripts/main.js#L19 

```js
$window.on('load', function() {
  setTimeout(function() {
    $('body').removeClass('is-loading');
  }, 100);
});
```

Reply:
@thiamsantos yes! I think this could work!

Reply:
@tajchumber I pushed the changes, could you take another look?